src/R/Components/Impl/ConnectionManager/Implementation/ViewModel/ConnectionViewModel.cs

```diff
@@ -192,7 +194,7 @@ internal sealed class ConnectionViewModel : BindableBase, IConnectionViewModel {
 
             Uri uri;
             var isPathValid = Uri.TryCreate(Path, UriKind.Absolute, out uri);
-            if (string.IsNullOrEmpty(Name)) {
+            if (!IsValidConnectionName(Name)) {
```

Comment:
It fails to start the process, throws in n`ew FileLoggerProvider(_startupOptions.Name)` so there is no connection to report the API error.